### ISSUE TITLE
SClientIntegrationSpec doesn't run twice

### ISSUE THREAD

**Issue by khlumzeemee** [jasongoodwin/learning-akka#8]
The default implementation of the `SClientIntegrationSpec` class gives an error if we try to run the tests twice. (port already in use).

The default template from giter8 uses `TestKit` and `BeforeAndAfterAll` to cleanly close the test class after execution. Why not use those?

**Comment by jasongoodwin** [comment]
Yep you're correct. Someone else pointed this out as well - I just haven't gotten around to fixing it. Feel free to make a pull request if you have a good example already.
